src/components/NavBar/NavBar.scss

```diff
@@ -0,0 +1,15 @@
+.navbar{
+    display: flex;
+    height: 30px;
+    background-color: #bcda12;
+    align-items: center;
+
+    a{
+        color: black;
```

Comment:
how about we choose one color format , Hex color ?

Reply:
i agree, will make the change

Reply:
Please review the change and resolve the comment if you deem it solved